app/jobs/create_immunity_details_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateImmunityDetailsJob < ApplicationJob
+  queue_as :urgent
```

Comment:
Changing this to `high_priority` with the [sidekiq](https://github.com/unboxed/bops/pull/1030) changes so just a heads up depending on which one gets merged first